corehq/apps/app_manager/models.py

```diff
@@ -5386,17 +5388,13 @@ def fetch_jar(self):
 
     def make_build(self, comment=None, user_id=None, previous_version=None):
         copy = super(ApplicationBase, self).make_build()
+        copy._all_files = self._all_files
```

Comment:
Is the object referenced by `self._all_files` mutable (I think it's a dict, which would mean yes)? If yes, is it ever mutated? Would it make sense to make a (probably shallow) copy here rather than assigning the same reference so mutations in this app do not affect the build (or the reverse)?

Reply:
```self._all_files``` is never mutated.  (It is a ```dict```, so it is a mutable object.)

Right now, it's a private field, so I'm not particularly worried about ```self``` and ```copy``` sharing the same object.